src/routes/results/[results_string]/+page.svelte

```diff
@@ -114,7 +131,15 @@
 		{/each}
 		<path stroke-width="3" stroke="navy" fill="navy" opacity="0.8" d={drawAnswerShape(answers)} />
 		{#each formattedAnswers as ans}
-			<circle cx={ans.xCoord} cy={ans.yCoord} fill="navy" r={config.answerR}></circle>
+			<circle
+				on:mouseover={() => mouseOver(dynamics[ans.idx])}
+				on:mousemove={(evt) => mouseMove(evt)}
```

Comment:
nit: this line can be simplified to 
```suggestion
				on:mousemove={mouseMove}
```
instead of the current anonymous function, since it's just being called with the event